src/Applications/LDAS_App/preprocess_ldas_routines.F90

```diff
@@ -2152,32 +2153,53 @@ subroutine optimize_latlon(fname,arg)
           n2 = k*IMGLOB
           
           do i=1,60
-             rates(i) = (i-1)*0.1
+             rates(i) = -0.3 + i*0.01
```

Comment:
Are these changes related to the "painter's partition" algorithm?  If so, we should probably add a comment somewhere that the distribution implemented here follows this algorithm.

Reply:
This is another effort to optimize the distribution. It is supposed to be better but not really as optimal as painter's partition algorithm.